java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1604,7 +1474,7 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_bitwiseMergeAndSetValidit
     std::transform(n_cudf_columns.data(), n_cudf_columns.data() + n_cudf_columns.size(),
                    std::back_inserter(column_views),
                    [](auto const &p_column) { return *p_column; });
-    cudf::table_view *input_table = new cudf::table_view(column_views);
+    cudf::table_view *input_table = new cudf::table_view(column_views); // <<---- TODO: MEMORY LEAK!
```

Comment:
Isn't this as simple as making this a `cudf::table_view` local variable rather than trying to allocate a table view on the heap?

Reply:
Yep. I haven't fixed this one yet. I'm only halfway through this file, so far.

Reply:
This file has now been dealt with.